crates/re_space_view/src/data_query_blueprint.rs

```diff
@@ -450,9 +478,10 @@ mod tests {
             all_recordings: vec![],
         };
 
-        let scenarios: Vec<(Vec<&str>, Vec<&str>)> = vec![
+        let scenarios: Vec<(Vec<&str>, Vec<&str>, Vec<&str>)> = vec![
```

Comment:
nit: A comment that this is (inclusions, exclusions, expected) would help!

Reply:
Turned into a struct to make it even clearer